website/docs/language/resources/ephemeral/write-only.mdx

```diff
@@ -65,35 +65,23 @@ ephemeral "random_password" "db_password" {
   override_special = "!#$%&*()-_=+[]{}<>:?"
 }
 
-resource "aws_secretsmanager_secret" "db_password" {
-  name = "db_password"
-}
-
-resource "aws_secretsmanager_secret_version" "db_password" {
-  secret_id                = aws_secretsmanager_secret.db_password.id
-  secret_string_wo         = ephemeral.random_password.db_password.result
-  secret_string_wo_version = 1
-}
-
-ephemeral "aws_secretsmanager_secret_version" "db_password" {
-  secret_id = aws_secretsmanager_secret_version.db_password.secret_id
-}
-
 resource "aws_db_instance" "example" {
   instance_class      = "db.t3.micro"
   allocated_storage   = "5"
   engine              = "postgres"
   username            = "example"
   skip_final_snapshot = true
-  password_wo         = ephemeral.aws_secretsmanager_secret_version.db_password.secret_string
-  password_wo_version = aws_secretsmanager_secret_version.db_password.secret_string_wo_version
+  password_wo         = ephemeral.random_password.db_password.secret_string
+  password_wo_version = 1
 }
 ```
 
 </CodeBlockConfig>
 
 During a Terraform operation, the provider uses the `password_wo` value to create the database instance, and then Terraform discards that value without storing it in the plan or state file. 
 
+Note that the way this is written, the `password_wo` value is lost after Terraform generates unless we capture it in another resource or output. For example of storing and retrieving an ephemeral password, Refer to the [expanded example below](#example). 
```

Comment:
Made the above example simpler and moved the full example to a separate section below. I love the point you make about the generated password being lost, so wanted to raise that for users too!